query.go

```diff
@@ -0,0 +1,78 @@
+package beacon
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"cloud.google.com/go/bigquery"
+)
+
+type Query struct {
+	refName string
+	allele  string
+	coord   *int64
+}
+
+func (q *Query) Execute(ctx context.Context, projectID, tableID string) (bool, error) {
+	query := fmt.Sprintf(`
+		SELECT count(v.reference_name) as count
+		FROM %s as v
+		WHERE %s
+		LIMIT 1`,
+		fmt.Sprintf("`%s`", tableID),
+		q.whereClause(),
+	)
+
+	bqClient, err := bigquery.NewClient(ctx, projectID)
```

Comment:
just 'client' since we don't need to distinguish

Reply:
done